x-pack/plugins/ml/public/components/entity_cell/entity_cell.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiToolTip
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+
+function getAddFilter({ entityName, entityValue, filter }) {
+  return (
+    <EuiToolTip
+      content={<FormattedMessage
+        id="xpack.ml.anomaliesTable.entityCell.addFilterTooltip"
+        defaultMessage="Add filter"
+      />}
+    >
+      <EuiButtonIcon
+        size="xs"
+        className="filter-button"
+        onClick={() => filter(entityName, entityValue, '+')}
+        iconType="plusInCircle"
+        aria-label={i18n.translate('xpack.ml.anomaliesTable.entityCell.addFilterAriaLabel', {
+          defaultMessage: 'Add filter'
+        })}
+      />
+    </EuiToolTip>
+  );
+}
+
+function getRemoveFilter({ entityName, entityValue, filter }) {
+  return (
+    <EuiToolTip
+      content={<FormattedMessage
+        id="xpack.ml.anomaliesTable.entityCell.removeFilterTooltip"
+        defaultMessage="Remove filter"
+      />}
+    >
+      <EuiButtonIcon
+        size="xs"
+        className="filter-button"
+        onClick={() => filter(entityName, entityValue, '-')}
+        iconType="minusInCircle"
+        aria-label={i18n.translate('xpack.ml.anomaliesTable.entityCell.removeFilterAriaLabel', {
+          defaultMessage: 'Remove filter'
+        })}
+      />
+    </EuiToolTip>
+  );
+}
+
+/*
+ * Component for rendering an entity in the anomalies table, displaying the value
```

Comment:
You should probably remove the reference to the anomalies table here, since it is now generic enough to be used in other places, like the Top Influencer list.

Reply:
Will update in this PR since I'll still be pushing to it: https://github.com/elastic/kibana/pull/33753